pkg/controller/master/backup/backupTaget.go

```diff
@@ -267,7 +267,8 @@ func (h *TargetHandler) validateS3BackupTarget(target *settings.BackupTarget) er
 	endpointResolver := aws.EndpointResolverFunc(func(service, region string) (aws.Endpoint, error) {
 		if target.Endpoint != "" {
 			return aws.Endpoint{
-				URL: target.Endpoint,
+				URL:           target.Endpoint,
+				SigningRegion: target.BucketRegion,
```

Comment:
What's the difference between `SigningRegion` and `Region` in the Endpoint fields?

Reply:
both are used for the region, but when the custom `endpointResolver` function is defined it will overwrite both the endpoint and region value, so in previous the `SigningRegion` is not defined and it will throw the `non-empty region` error.